cmake/modules/FindLIQ.cmake

```diff
@@ -28,6 +28,10 @@ IF (LIQ_FOUND)
   IF (NOT LIQ_FIND_QUIETLY)
     MESSAGE(STATUS "Found LIQ: ${LIQ_LIBRARY} ${LIQ_INCLUDE_DIR}")
   ENDIF (NOT LIQ_FIND_QUIETLY)
+ELSE (LIQ_FOUND)
+  IF (LIQ_FIND_REQUIRED)
+	MESSAGE(FATAL_ERROR "Could not find LIQ library")
+  ENDIF (LIQ_FIND_REQUIRED)
```

Comment:
Most standard `FindXXX.cmake` cmake module call `FIND_PACKAGE_HANDLE_STANDARD_ARGS` to throw exception when module is not found. The `IF` branch will not terminate cmake process. So I am considering update this code block to `FIND_PACKAGE_HANDLE_STANDARD_ARGS`.